src/pybind/mgr/dashboard/openapi.yaml

```diff
@@ -3711,26 +3709,37 @@ paths:
             trace.
       security:
       - jwt: []
-      summary: Get Ceph Users
+      summary: Get list of ceph users and its respective data
       tags:
       - Cluster
     post:
-      description: "\n        Add a ceph user with its defined capabilities.\n   \
-        \     :param user_entity: Entity to change\n        :param capabilities: List\
-        \ of capabilities to add to user_entity\n        "
```

Comment:
I still think that we should keep the method description (not the param).

Reply:
Re line 3712: s/respective/associated/

Reply:
done!